tests/models/concept_model_tests.py

```diff
@@ -292,3 +292,96 @@ def test_get_label_no_exact_match(self):
         self.assertEqual(pl.type, "prefLabel")
         self.assertEqual(pl.value, "bier" or "beer")
         self.assertEqual(pl.language, "nl" or "es-SP")
+
+    def test_sort_subconcepts_with_special_floats(self):
+        """
+        Test sorting subconcepts with special float-like values
+        """
```

Comment:
I'd remove a docstring that repeats the test title (thanks for adding a descriptive test title)